terracumber_config/tf_files/PR-TEST-main.tf

```
@@ -0,0 +1,218 @@
+        mac = var.ENVIRONMENT_CONFIGURATION[var.ENVIRONMENT].mac["suse-minion"]
+        vcpu = 2
+        memory = 2048
+      }
+      additional_repos = local.additional_repos["suse-minion"]
+      additional_packages = [ "venv-salt-minion" ]
+      install_salt_bundle = true
+    }
+    suse-sshminion = {
+      image = var.SUSE_MINION_IMAGE
+      name = "minssh-${var.SUSE_MINION_NAME}"
+      provider_settings = {
+        mac = var.ENVIRONMENT_CONFIGURATION[var.ENVIRONMENT].mac["suse-sshminion"]
+        vcpu = 2
+        memory = 2048
+      }
+      additional_repos = local.additional_repos["suse-minion"]
+      additional_packages = [ "venv-salt-minion", "iptables" ]
+      install_salt_bundle = true
+    }
+    redhat-minion = {
+      image = var.REDHAT_MINION_IMAGE
+      name = "min-${var.REDHAT_MINION_NAME}"
+      provider_settings = {
+        mac = var.ENVIRONMENT_CONFIGURATION[var.ENVIRONMENT].mac["redhat-minion"]
+        memory = 2048
+        vcpu = 2
+      }
+      additional_repos = {
+        client_repo = var.RHLIKE_CLIENT_REPO,
+      }
+      additional_packages = [ "venv-salt-minion" ]
+      install_salt_bundle = true
+    }
+    debian-minion = {
+      image = "ubuntu2204o"
+      name = "min-ubuntu2204"
+      provider_settings = {
+        mac = var.ENVIRONMENT_CONFIGURATION[var.ENVIRONMENT].mac["debian-minion"]
+        vcpu = 2
+        memory = 2048
+      }
+      additional_repos = {
+        client_repo = var.DEBLIKE_CLIENT_REPO,
+      }
+      additional_packages = [ "venv-salt-minion" ]
+      // FIXME: cloudl-init fails if venv-salt-minion is not avaiable
+      // We can set "install_salt_bundle = true" as soon as venv-salt-minion is available Uyuni:Stable
+      install_salt_bundle = true
+    }
+    build-host = {
+      image = "sles15sp4o"
+      name = "min-build"
+      provider_settings = {
+        mac = var.ENVIRONMENT_CONFIGURATION[var.ENVIRONMENT].mac["build-host"]
+        vcpu = 4
+        memory = 8192
+      }
+      additional_repos = {
+        tools_update_pr = var.SLE_CLIENT_REPO
+      }
+      additional_packages = [ "venv-salt-minion" ]
+      install_salt_bundle = true
+    }
+    pxeboot-minion = {
+      image = "sles15sp4o"
+      provider_settings = {
+        vcpu = 2
+        memory = 2048
+      }
+      additional_repos = {
+        tools_update_pr = var.SLE_CLIENT_REPO
+      }
+      additional_packages = [ "venv-salt-minion" ]
+      install_salt_bundle = true
+    }
+    kvm-host = {
+      image = var.IMAGE
+      name = "min-kvm"
+      additional_grains = {
+        hvm_disk_image = {
+          leap = {
+            hostname = "suma-pr${var.ENVIRONMENT}-min-nested"
+            image = "http://${var.DOWNLOAD_ENDPOINT}/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-JeOS.x86_64-OpenStack-Cloud.qcow2"
+            hash = "http://${var.DOWNLOAD_ENDPOINT}/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-JeOS.x86_64-OpenStack-Cloud.qcow2.sha256"
```

```suggestion
            image = "http://${var.DOWNLOAD_ENDPOINT}/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-Minimal-VM.x86_64-OpenStack-Cloud.qcow2"
            hash = "http://${var.DOWNLOAD_ENDPOINT}/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-Minimal-VM.x86_64-OpenStack-Cloud.qcow2.sha256"
```
Is the new name of the image. It got renamed upstream, and the old one will get removed from the mirror, soon. 
See 
- http://minima-mirror.mgr.suse.de/distribution/leap/15.4/appliances/
- https://github.com/uyuni-project/sumaform/pull/1360